packages/ERTP/src/amountMath.js

```diff
@@ -194,47 +220,124 @@ const AmountMath = {
     // Will throw on inappropriate value
     return AmountMath.make(brand, allegedValue);
   },
+  /**
+   * Extract and return the value.
+   *
+   * @template {AmountValue} [V=AmountValue]
+   * @param {Brand} brand
+   * @param {Amount<V>} amount
+   * @returns {V}
+   */
   getValue: (brand, amount) => AmountMath.coerce(brand, amount).value,
+  /**
+   * Return the amount representing an empty amount. This is the
+   * identity element for MathHelpers.add and MatHelpers.subtract.
```

Comment:
I'm not sure `MathHelpers` are part of the external API.

Reply:
I just copied this from the earlier doc